src/internal/connector/exchange/exchange_data_collection.go

```diff
@@ -58,6 +59,7 @@ type Collection struct {
 
 	// service - client/adapter pair used to access M365 back store
 	service graph.Servicer
```

Comment:
can this be removed?

Reply:
In the future, yes.  Currently it's in use here: https://github.com/alcionai/corso/pull/2008/files#diff-560fe38c88a98120137bda62f4b736462f74b1549e7565578f6d03ad5e8cd6a3L286